### ISSUE TITLE
can't no find  contentType

### ISSUE THREAD

**Issue by yzzshmily** [gmuth/ipp-samples#1]
I used   HP PageWide MFP P57750 - J9V82D run  this  java samples  ，  can't find  contentType in IppClient.kt:102 ,why?

what should I do?

```
Exception in thread "main" java.lang.NullPointerException
	at de.gmuth.ipp.client.IppClient.exchange(IppClient.kt:102)
	at de.gmuth.ipp.client.IppClient.exchangeSuccessful(IppClient.kt:67)
	at de.gmuth.ipp.client.IppPrinter.exchangeSuccessful(IppPrinter.kt:250)
	at de.gmuth.ipp.client.IppPrinter.getPrinterAttributes(IppPrinter.kt:140)
	at de.gmuth.ipp.client.IppPrinter.getPrinterAttributes$default(IppPrinter.kt:138)
	at de.gmuth.ipp.client.IppPrinter.updateAllAttributes(IppPrinter.kt:144)
	at de.gmuth.ipp.client.IppPrinter.<init>(IppPrinter.kt:28)
	at de.gmuth.ipp.client.IppPrinter.<init>(IppPrinter.kt:24)
	at de.gmuth.ipp.client.IppPrinter.<init>(IppPrinter.kt:37)
	at ipp.PrinterAttributes.main(PrinterAttributes.java:9)
```

**Comment by gmuth** [comment]
I've updated the ipp-client-kotlin lib to throw another exception when no content-type is available.
usually this is the case when the printer-uri is wrong.
- what printer-uri do you use?
- do you use maven or gradle? (please use the head of branch master of ipp-client-kotlin)

also please provide more log output.

**Comment by gmuth** [comment]
NullPointerException has been fixed in [ipp-client-kotlin release 2.1](https://github.com/gmuth/ipp-client-kotlin/releases/tag/2.1)
